quick-start/src/main/ui/app/shared/stories/components/select/select.component.stories.ts

```diff
@@ -0,0 +1,99 @@
+import {moduleMetadata, storiesOf} from '@storybook/angular';
+import {object, text, boolean, withKnobs} from '@storybook/addon-knobs';
+import {withNotes} from '@storybook/addon-notes';
+import {action} from '@storybook/addon-actions';
+import {linkTo} from '@storybook/addon-links';
+import {centered} from '@storybook/addon-centered/angular';
+
+import {SelectComponent} from '../../../components/select/select.component';
+import {ThemeModule} from '../../../components/theme/theme.module';
+import {Component, Input} from '@angular/core';
+
+@Component({
+    selector: 'mlui-story-card',
+    template: `
+        <div class="container">
+            <div class="card story-card" [style.width]="width + 'px'" [style.height]="height + 'px'">
+                <ng-content></ng-content>
+            </div>
+        </div>
+    `,
+    styles: [`
+        .story-card {
+            padding: 10px 10px 10px 10px;
+            box-shadow: 10px 10px 20px;
+        }
+    `]
+})
+class StoryCardComponent {
+
+    @Input() width = 500;
+    @Input() height = 200;
+
+    constructor() {}
+}
```

Comment:
StoryCardComponent exists on ../../utils/ level. Please use the one from there. See in Scott's PR how it is done.